.github/workflows/tmt-tests.yml

```diff
@@ -11,7 +11,7 @@ jobs:
     name: Run tmt tests on Testing Farm service
     if: |
       github.event.issue.pull_request
-      && github.event.comment.body == '/rerun'
+      && startsWith(github.event.comment.body, '/rerun')
```

Comment:
no spaces allowed huh?

Reply:
TBH I didn't give it much thought though, it's how it has always been with our comment-based commands.
How about I fix that in a separate patch? So that I could simultaneously do that for both leapp and leapp-repository?

Reply:
It wasn't really a 'Hey you need to fix that', just more like I didn't even realize. But TBH I am fine with it as is.